NEXT_ENV_FIX.md

```
@@ -0,0 +1,170 @@
+
+Example scenario:
+
+```typescript
+// next-env.d.ts  ← DO NOT EDIT
+declare const __COMMIT__: string;  // ← added by a developer
+```
+
+After the next `pnpm dev` or `pnpm build`, the file is regenerated. The declaration is
+gone. TypeScript then fails with:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)